app/models/indicator.rb

```diff
@@ -142,6 +142,11 @@ def added_by_order_clause(order_direction)
           WHEN variations.id IS NOT NULL THEN variations_teams.name
           WHEN indicators.model_id IS NOT NULL THEN teams.name
           ELSE NULL
+        END,
```

Comment:
The lack of a second order criterion here has been making the specs fail randomly, which we had endured for a while now and I decided today is the day to get rid of it :)